inst/workflow/1_mappings/STUDY.yaml

```diff
@@ -41,6 +42,8 @@ spec:
       type: character
     subjid:
       type: character
+    Status:
```

Comment:
Another duplicate Status as far as i can tell

Reply:
Nice catch!

Reply:
Oh wait, `Mapped_SUBJ$Status` is used to calculate the number of active subjects.  `Raw_STUDY$Status` is a study-level attribute.